cmd/client/status.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/monstercat/gologx/logxhost"
+	"github.com/olekukonko/tablewriter"
+)
+
+func showStatus(name string, args []string) error {
+	var postgres string
+	var services string
+
+	set := flag.NewFlagSet(name, flag.ExitOnError)
+	set.StringVar(&postgres, "postgres", "", "Postgres database")
+	set.StringVar(&services, "services", "", "List of services")
+
+	if err := set.Parse(args); err != nil {
+		return err
+	}
+
+	db, err := logxhost.GetPostgresConnection(postgres)
+	if err != nil {
+		return err
+	}
+	defer db.Close()
+
+	srvcsArr := func() []string {
+		if services != "" {
+			return strings.Split(strings.ReplaceAll(services, " ", ""), ",")
+		} else {
+			return make([]string, 0)
+		}
+	}()
```

Comment:
i think you are doing this because the default returns length 1 for empty string. Perhaps separate this out into another function under `golib` `stringutils` with a comment noting the difference in default.

Reply:
I am doing this because I need srvcsArr to be == [] if the string is equal "" => "" value is the default for when no params are passed to --services. I don't think that this should be standardized as a function because it is a specific case for this CLI tool, where we indicate that services can be passed as flags, and these must be comma-separated. 

In any other system, it could be spaces separated, etc, depending on the required string format. Don't see any use of it in a big scale at least yet.

Reply:
In that case perhaps we can separate this out into a private function in this package as it is used twice. 

Or you could pass in the separator as a parameter and still have it as a utility function.